comp/core/flare/flare.go

```diff
@@ -183,6 +183,7 @@ func (f *flare) Create(pdata ProfileData, ipcError error) (string, error) {
 		},
 		f.collectLogsFiles,
 		f.collectConfigFiles,
+		f.collectHostSBOM,
```

Comment:
Please use a [flare provider](https://datadoghq.dev/datadog-agent/components/shared_features/flares/) instead.

Reply:
Does it imply creating a component ?

Reply:
I'm not sure to understand. I added my code to comp/core/flare/helpers/providers.go, as it's done for log files and config files. Can't it be a new callback ?